website/docs/r/logic_app_integration_account_batch_configuration.html.markdown

```diff
@@ -0,0 +1,125 @@
+---
+subcategory: "Logic App"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_logic_app_integration_account_batch_configuration"
+description: |-
+  Manages a Logic App Integration Account Batch Configuration.
+---
+
+# azurerm_logic_app_integration_account_batch_configuration
+
+Manages a Logic App Integration Account Batch Configuration.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_logic_app_integration_account" "example" {
+  name                = "example-ia"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku_name            = "Standard"
+}
+
+resource "azurerm_logic_app_integration_account_batch_configuration" "example" {
+  name                     = "example-iabc"
+  resource_group_name      = azurerm_resource_group.example.name
+  integration_account_name = azurerm_logic_app_integration_account.example.name
+  batch_group_name         = "TestBatchGroup"
+
+  release_criteria {
+    message_count = 80
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Logic App Integration Account Batch Configuration. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Logic App Integration Account Batch Configuration should exist. Changing this forces a new resource to be created.
+
+* `integration_account_name` - (Required) The name of the Logic App Integration Account. Changing this forces a new resource to be created.
+
+* `batch_group_name` - (Required) The batch group name of the Logic App Integration Batch Configuration. Changing this forces a new resource to be created.
+
+* `release_criteria` - (Required) A `release_criteria` block as documented below.
```

Comment:
what does release criteria mean? could we document this?

Reply:
Updated. Found the description from https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-batch-process-send-receive-messages#create-batch-receiver